datafusion/sql/src/unparser/expr.rs

```diff
@@ -992,7 +1060,11 @@ mod tests {
                     ),
                     args: vec![wildcard()],
                     partition_by: vec![],
-                    order_by: vec![],
+                    order_by: vec![Expr::Sort(Sort::new(
+                        Box::new(col("a")),
+                        false,
+                        true,
+                    ))],
```

Comment:
this attaches the ordering into the aggregate function arguments -- I think it needs to be some part of `WindowFrame`